pkg/controllers/cluster/reference.go

```diff
@@ -71,7 +71,7 @@ func (h *handler) claimCluster(cluster *v1.Cluster, status v1.ClusterStatus) (*v
 		}
 		updated.Labels[claimedLabelName] = cluster.Name
 		updated.Labels[claimedLabelNamespace] = cluster.Namespace
-		return h.rclusters.Update(updated)
```

Comment:
Oh so did I miss in this? and that's why it was not working?

Reply:
Yes, everything you did worked, but there was one more spot to use it.

Reply:
:+1: